### DISCUSSION TITLE
`internal bridge infill` above infill pattern printing onto empty space and curling

### DISCUSSION THREAD

**Discussion by kevenwyld** [supermerill/SuperSlicer#3117] [Q&A] [answered]
How do I prevent curling when using `internal bridge infill` above gyroid pattern infill for example?  The `internal bridge infill` prints a layer on top of the `internal infill` but there isn't necessarily any internal infill underneath it.  Is there some way to configure that first `internal bridge infill` to extend out to the nearest line from the previous infill layer so that it is supported?

![Screenshot_2022-08-22_15-14-56](https://user-images.githubusercontent.com/11936535/186020685-c8324815-4c3a-4937-aa35-fcb93cd0be79.jpg)

![IMG_20220822_150802](https://user-images.githubusercontent.com/11936535/186020720-73d7943e-ed26-4098-a193-f3f15627ae74.jpg)
![IMG_20220822_150718](https://user-images.githubusercontent.com/11936535/186020722-d6bb2eae-ce4a-467f-93f6-60cf69c7e8a2.jpg)

EDIT: Sorry, I don't use discussions much.  This works better in the main post than as a comment.  Here's the zipped 3mf file if anyone wants to take a look:
[SHORT_THROW_SHIFTER_flatened_for_github.3mf.zip](https://github.com/supermerill/SuperSlicer/files/9397686/SHORT_THROW_SHIFTER_flatened_for_github.3mf.zip)

**Comment by neophyl** [comment, marked as answer]
Use the supporting dense layer feature Infill>Reducing Print Time>Supporting Dense Layer
[SHORT_THROW_SHIFTER_flatened_for_github_neo.zip](https://github.com/supermerill/SuperSlicer/files/9399803/SHORT_THROW_SHIFTER_flatened_for_github_neo.zip)

**Reply by kevenwyld** [reply]
Thanks!  That's exactly what I was looking for.  For this part I think the full option will work best.

![Screenshot_2022-08-23_07-57-02](https://user-images.githubusercontent.com/11936535/186177300-009f6738-484b-4a15-8c6f-08ac9c64cb32.jpg)

![Screenshot_2022-08-23_07-55-24](https://user-images.githubusercontent.com/11936535/186176903-f77f3636-a1c4-4303-8257-0484572842c9.jpg)
